package.json

```diff
@@ -2,47 +2,52 @@
   "name": "@mswjs/interceptors",
   "description": "Low-level HTTP/HTTPS/XHR/fetch request interception library.",
   "version": "0.19.3",
-  "main": "./lib/node/index.mjs",
+  "main": "./lib/node/index.js",
+  "module": "./lib/node/index.mjs",
   "types": "./lib/node/index.d.ts",
   "exports": {
     ".": {
       "types": "./lib/node/index.d.ts",
-      "node": "./lib/node/index.js",
+      "default": "./lib/node/index.js",
       "require": "./lib/node/index.js",
-      "default": "./lib/node/index.mjs"
+      "import": "./lib/node/index.mjs"
```

Comment:
I think we need default to come last - these are matched in order from top to bottom, so default will always beat out require/import in this case (and might throw in some tools?)

Reply:
Didn't know about that. Thanks, @mattcosta7. Do you happen to know where this order is defined? I didn't see it in the Node's ESM docs.

Reply:
Within the "exports" object, key order is significant. During condition matching, earlier entries have higher priority and take precedence over later entries. The general rule is that conditions should be from most specific to least specific in object order.

https://nodejs.org/api/packages.html#conditional-exports

Reply:
Fantastic. Thank you!